lib/web_ui/dev/chrome.dart

```diff
@@ -82,10 +82,8 @@ class Chrome extends Browser {
       final String dir = environment.webUiDartToolDir.createTempSync('test_chrome_user_data_').resolveSymbolicLinksSync();
       final String jsFlags = enableWasmGC ? <String>[
         '--experimental-wasm-gc',
-        '--wasm-gc-js-interop',
         '--experimental-wasm-stack-switching',
         '--experimental-wasm-type-reflection',
-        '--wasm-gc-js-interop',
```

Comment:
Lol, why did I put this flag in here twice

Reply:
lol